File: chapter6/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    
    do
    {
        printf("Height: ");
        height = GetInt();
    } while (height < 0 || height > 23);
    
    for (int i = 0; i < height; i++)
    {
        for (int j = 1; j < height - i; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 2; j++)
        {
            printf("#");
        }    
        printf("\n");
    }
}

File: chapter6/mario.py
import cs50

while True:
    print("Height: ", end="")
    height = cs50.get_int()
    if 0 <= height <= 23:
        break
    
for i in range(height):
    for j in range(1,height-i):
        print(" ", end="")
    for j in range(i+2):
        print("#", end="")
    print()